plasma_framework/test/src/exits/payment/PaymentExitGame.test.js

```diff
@@ -1,12 +1,17 @@
+const EthVault = artifacts.require('EthVault');
+const Erc20Vault = artifacts.require('Erc20Vault');
+const EthDepositVerifier = artifacts.require('EthDepositVerifier');
+const Erc20DepositVerifier = artifacts.require('Erc20DepositVerifier');
+const ExitId = artifacts.require('ExitIdWrapper');
+const GoodERC20 = artifacts.require('GoodERC20');
```

Comment:
I think `malicious` is a bit weird here too. They are not `malicious` (otherwise OMG would be one of the `malicious` ones, oops)
But I agree `Good` and `Bad` sounds naive. Let's see if we can reach some consensus here, but I would prefer to do that in a separate PR to make code review easier as that's a minor issue of this PR?